user_guide/content/modules/creating_a_new_space.adoc

```diff
@@ -11,28 +11,29 @@ To start a new project, users must create a new Collaboration Space. Each Space
 
 . In your {osio-link} home page, click your username in the top left corner of the screen.
 
-. In the drop down menu, click *Create space*.
+. In the drop-down menu, click *Create space*.
 
 . Add a unique name for the new space in the *Name* field.
 
-. Select a *Template* type from the drop-down menu. The four options are:
-.. *Agile* - Your space is centered around Agile-based plannning.
-.. *Scrum* - Your space includes an iterative and incremental Agile-based development framework. This option is similar to the **Agile** option but with more specific planning and development tracking.
-.. *Issue Tracking* - Your space is based on an issue tracking development methodology. This option is ideal for teams interested in primarily tracking issues and Tasks and resolving them.
-.. *Scenario Driven Planning* - Your space is based on Scenarios, which are real-world problems from the user's point of view. The Scenarios deliver Value Propositions and are realized via (user) Experiences.
+. Select *Scenario Driven Planning* from the drop-down menu for templates. This templates creates Spaces based on Scenarios, which are real-world problems from the user's point of view. The Scenarios deliver Value Propositions and are realized based on (user) Experiences.
+//. Select a *Template* type from the drop-down menu. The available options are:
```

Comment:
templates>template
I am not sure about 'based on' user experience, wondering if it changes the meaning. I thought 'through' was a good alternative. Probably Robert would be able to add some light to it.